src/services/homework/hooks/index.js

```diff
@@ -113,7 +116,7 @@ const addStats = (hook) => {
 			// save grade in assignment if user is student of this task
 			const submission = submissions.data.filter((s) => (equalIds(c._id, s.homeworkId) && (s.grade)));
 			if (submission.length == 1 && !isTeacher(hook.params.account.userId, c)) {
-				c.grade = submission[0].grade;
+				c.grade = submission[0].grade.toFixed(2);
```

Comment:
could this be a helper or a getter on the model?

Reply:
I guess it could, but that would probably confuse other services that want to treat the grade as number to compute stuff like averages (something the same hook does a few lines below)